src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -95,19 +96,19 @@
 
 if (\PHP_VERSION_ID >= 80000) {
     // PHP 8 requires PHPUnit 9.3+
-    $PHPUNIT_VERSION = $getEnvVar('SYMFONY_PHPUNIT_VERSION', '9.4');
+    $PHPUNIT_VERSION = $getEnvVar('SYMFONY_PHPUNIT_VERSION', '9.4', true);
```

Comment:
this would do it and would be easier to grasp I think:
```diff
-$PHPUNIT_VERSION = $getEnvVar('SYMFONY_PHPUNIT_VERSION', '9.4', true)
+$PHPUNIT_VERSION = $getEnvVar('SYMFONY_PHPUNIT_VERSION', '9.4') ?: '9.4'
```

Reply:
is fine for me, did change it.